source/usage-examples/distinct.txt

```diff
@@ -26,8 +26,8 @@ Click `here <{+example+}/distinct.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-The example returns an empty slice of an ``interface`` type that
-contains the following values:
+After you run the preceding example, it returns an empty slice of an
+``interface`` type that contains the following values:
```

Comment:
Question: I'm confused what is returned. If it's an empty slice, why does it contain values? Or are you trying to say that the interface contains values? I would reword this to clarify this ambiguity.

Reply:
So in Go, an [empty interface](https://tour.golang.org/methods/14) is a term used to describe `interface{}` regardless of how many documents it contains.